File: src/app/feedback-dto.ts
export class FeedbackDTO {

    feedbackId:number;
    patientName:string;
    hospitalName:string;
    doctorName:string;
    rating:number;
    feedback:string;


    constructor(
        feedbackId:number,
        patientName:string,
        hospitalName:string,
        doctorName:string,
        rating:number,
        feedback:string
    ){
        this.feedbackId=feedbackId;
        this.patientName=patientName;
        this.hospitalName=hospitalName;
        this.doctorName=doctorName;
        this.rating=rating;
        this.feedback=feedback;
    }

}

File: feedback/FeedbackDTO.java
package com.nv.doctorapp.dto.feedback;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class FeedbackDTO {

	private int feedbackId;
	private String patientName;
	private String doctorName;
	private String hospitalName;
	private int rating;
	private String feedback;
}
